### ISSUE TITLE
[BarViz] Accessibility for Blazor front-end

### ISSUE THREAD

**Issue by premun** [dotnet/arcade-services#4048]
We need to run/fix accessibility scans (using the Accessibility Inisghts extension) on the new front-end.

We also need to give instructions to the team performing the manual A11Y checks (which means they need to be able to log in) and we need to process any bugs they find.

**Comment by oleksandr-didyk** [comment]
On top of the PR linked above, we would need an update for the `FluentUI` library to deal with a minor accessibility issue with an element that should not be keyboard accessible. The [fix](https://github.com/microsoft/fluentui-blazor/issues/2901) we required should be in version `4.10.4`

**Comment by oleksandr-didyk** [comment]
> The fix we required should be in version 4.10.4

Will keep the issue opened until we upgrade to the required version

**Comment by oleksandr-didyk** [comment]
Fluent UI library was updated and the expected fix verified
